docker.Makefile

```diff
@@ -121,7 +121,7 @@ save-invocation-image: invocation-image
 
 load-invocation-image:
 	docker load -i $(CNAB_BASE_INVOCATION_IMAGE_PATH)
-	$(RM) $(CNAB_BASE_INVOCATION_IMAGE_PATH)
+#	$(RM) $(CNAB_BASE_INVOCATION_IMAGE_PATH)
```

Comment:
This will leak files into /tmp or similar. Really that file should not be put there in the first place -- it should be loaded into the workspace somewhere and this ought to be fixed IMHO. As it stands even without commenting out this line it will leak if `docker load` fails for some reason.

Reply:
That was just a test... That PR is still WIP